python/lancedb/context.py

```diff
@@ -140,6 +140,14 @@ def text_col(self, text_col: str) -> Contextualizer:
     def to_df(self) -> pd.DataFrame:
         """Create the context windows and return a DataFrame."""
 
+        if self._window is None:
+            raise MissingValueError("The value of window is None. Specify the" + \
```

Comment:
No backslash or plus needed here in Python
```suggestion
            raise MissingValueError("The value of window is None. Specify the"
```
https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation